resources/keywords/suite_and_test_setup_teardown.resource

```diff
@@ -100,16 +114,18 @@ Setup The Test Case
     ...    AND    Set Global Variable    ${DBaaSPolicyConfigured}    ${True}
 
 Configure DBaaS Policy Namespaces
+    [Arguments]     ${ns}=${DBaaSTestNS}
```

Comment:
Is ${DBaaSTestNS} = redhat.dbaas.operator?

Reply:
No, The assignment of variable is done on login.resource file.